src/Module/Settings/Display.php

```diff
@@ -297,19 +330,19 @@ private function getAvailableTimelines(int $uid, bool $only_network = false): ar
 		$timelines = [];
 
 		foreach ($this->timeline->getNetworkFeeds('') as $channel) {
-			$timelines[$channel->code] = $this->t('%s: %s', $channel->label, $channel->description);
+			$timelines[$channel->code] = ['label' => $channel->label, 'description' => $channel->description];
```

Comment:
Why don't you return the whole entity objects here?

Reply:
In 2 of the 3 occasions this function is called, I only need the array keys. If I returned some class here, then i had to add some additional coding to these places (instead of just using `array_keys`) to fetch the keys.

Reply:
What I mean is `$timelines[$channel->code] = $channel`, this way you can still use `array_keys` in the resulting calls. If you need the label and the description, you just have to use the class properties instead of the array keys you created.

Reply:
It is changed now.